src/lib.rs

```diff
@@ -19,3 +19,9 @@ pub const DEFAULT_EXPIRY_TIME: Duration = Duration::from_secs(70);
 pub const DEFAULT_EVENTS_TOPIC: &str = "wasmbus.evt.*";
 /// Default topic to listen to for all commands
 pub const DEFAULT_COMMANDS_TOPIC: &str = "wadm.cmd.*";
+/// Managed by annotation used for labeling things properly in wadm
+pub const MANAGED_BY_ANNOTATION: &str = "wasmcloud.com/managed-by";
```

Comment:
The old (current) wadm didn't need this. It just used the "spec" annotation to indicate which oam model owned a particular instance of a thing. If the spec didn't exist then it was considered unmanaged.

Is there a different consideration here that needs a managed by as well as the claimant spec?

Reply:
Oh I actually think that is a better idea. I'll go update

Reply:
Note that there is a specific annotation already detected by the lattice observer used by the current wadm.